.github/workflows/build-private-example-web.yml

```
@@ -0,0 +1,43 @@
+on: workflow_dispatch
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      TEST_ENDPOINT: hd-example.onekeytest.com
+      PROJECT_PATH: packages/connect-examples/expo-example
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Environment
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)